content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -466,43 +466,36 @@ To set a different service name per Ingress using annotations:
 ```
 The above overrides the default `nginx-ingress-controller.ingress-nginx` service name.
 
-### Sampling
+### Ingress Controller Sampling
 
-To control the volume of traces starting from Nginx that are sent to Datadog, specify a sampling rate by setting the parameter `DD_TRACE_SAMPLING_RULES` to a value between `0.0` (0%) and `1.0` (100%). If no value is specified, 100% of traces starting from Nginx are sent. The Kubernetes Nginx ingress controller uses [v1.2.1][11] of the `dd-opentracing-cpp` library).
```

Comment:
I would keep mentioning that the Ingress Controller uses the v1.2.1 of the opentracing library